phpunit.xml.dist

```diff
@@ -17,13 +17,17 @@
         <env name="TYPO3_CONSOLE_TEST_RUN" value="1" />
         <env name="TYPO3_INSTALL_DB_USER" value="root" />
         <env name="TYPO3_INSTALL_DB_HOST" value="127.0.0.1" />
-        <env name="TYPO3_INSTALL_DB_PASSWORD" value="" />
+        <env name="TYPO3_INSTALL_DB_PASSWORD" value="root" />
         <env name="TYPO3_INSTALL_DB_DBNAME" value="travis_console_test" />
```

Comment:
Does not match the name used below, is this intentional?

Reply:
I didn't pay attention to that. I needed to set the DB PASSWORD to be able to run test in GHA. Will check this out.